recipe/LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
```

Comment:
Why did you copy these files from repo root to inside recipe?

Reply:
@isuruf Thanks! I see what happened now. I inadvertently ran `conda smithy rerender` when I was in the recipe folder, realized my mistake (but didn't notice it added and commited a bunch of stuff to `recipe`), and then ran it again in the root folder. I'll remove the extra stuff.